src/nni_manager/common/log.ts

```diff
@@ -81,6 +82,7 @@ class Logger {
         }
 
         this.readonly = isReadonly();
+        this.foreground = foreground;
```

Comment:
Suggest to do a little bit refactor on log.ts:
currently fileName being undefined means to use nnimanager.log as output file. change it to:
1. undefined means send output to STDOUT. 
2. Move file name `nnimanager.log` into main.ts

Then we do not need a `foreground` flag for log.ts.  Because both log files and STDOUT are places where log class can send content to, we do not need an extract flag `foreground` for STDOUT from log class perspective.

Reply:
fixed.